apps/vc-api/docs/example-exchange-definitions/presentation-issuance-self-sign-exchange.md

```diff
@@ -0,0 +1,95 @@
+Sample Exchange Definition for a issuance, self-sign and presentation.
+
+Use case:
+
+1. The Holder is required to present a self-signed consent VC and a PermanentResidentCard VC (Issued by Authority) to obtain another Verifiable Credential from the Issuer.
+
+`PresentationDefinition` exchange Definition.
+
+``` json
```

Comment:
Some the exchanges have "`PresentationDefinition` exchange Definition.", which I think is left over from when there was `DIDAuth` as well. I propose removing this from all of the exchange definitions
```suggestion
``` json
```